### ISSUE TITLE
Website (Forkaweso.me) is down

### ISSUE THREAD

**Issue by ginsterbusch** [ForkAwesome/Fork-Awesome#226]
The website / server seems to be down. At least I can't send pings to it, nor run traceroutes.

**Comment by njacquemin** [comment]
Yes, during the last few days, the website was down, then online, then down...

**Comment by tessus** [comment]
I don't know who's the owner of that domain, web server, VM, ... @xuv any ideas?

**Comment by ginsterbusch** [comment]
Maybe having it kept at github.io would have been the smarter move?

Nearly got in a verbal fight over the phone with a colleague, who is very pro FrontAwesome, saying SUCH idioticies are the very reason he doesnt use Fork Awesome anymore (and then was stomping around on Open Source in general, until I pointed out that if there were no FOSS, most of those bits we call "the interwebz" wouldnt be there anyway).

cu, w0lf.

**Comment by tessus** [comment]
I don't know what prompted the move to the different domain. It might still be on github. I have no idea how it is setup. I always thought the domain is only an alias.
Anyway, since I didn't set it up, there's nothing I can do. I hope @xuv can shed some light on this.

**Comment by xuv** [comment]
Oh. Thanks all for the heads up. I'll have a look into it.

**Comment by xuv** [comment]
Hello all.

A bit of clarification and explanation, I forgot to pay the renewal of the domain. I indeed am the owner of the name. It's now renewed. Website should be accessible soon.

The reason I decided to use a custom domain name for this project is to gain independence from vendors such as Github. I appreciate them hosting this project for now. We could be tempted to move to a different provider some time in the future. Having a custom domain helps people finding us again in the future wherever we are.

About trust, I want to make clear that this domain name being down for a few days only impacted the web documentation of this project. The full source code was still available from here, on Github. The fonts, css, less and sass files were all still available from our different CDNs and NPM repositories. So, as far as I know, this had no impact on any of the people developing or using ForkAwesome anywhere.

Hope this clears out most of the questions and thank you for the care and attention you put into this project.

Julien
